src/utils/url.ts

```diff
@@ -98,12 +98,15 @@ export function getEntryUrlForPortal(
     throw new Error(`Given public key '${publicKey}' is not a valid hex-encoded string or contains an invalid prefix`);
   }
 
-  // We need to hash the data key in order to form the correct URL, as Sia hashes whatever data key we give it.
-  const dataKeyHash = toHexString(hashDataKey(dataKey));
+  // If not a hashed byte array already, hash the data key in order to form the correct URL.
```

Comment:
is this correct? Shouldn't it be `Hash and hex encode the given datakey if the given data key is not a hash already`?

Reply:
Good catch, pertained to a previous commit